### PR TITLE
Fix yes/no problem in presentation.clj

### PR DESCRIPTION
**Pull request by Ivanio1**
An error related to reading different values of yes and no was discussed in the chat. Corrected it. I made it so that the bot always accepts "yes" or "no", and not "да" or "нет", since this is how it is implemented in the other modules. Plus, the buttons in the bot are also set to "yes" or "no".

### CODE REVIEW

**Comment by Ivanio1** [comment]
@ryukzak

File: src/course_bot/presentation.clj [hunk 1]
```diff
@@ -115,8 +115,8 @@
     :drop-state-1 "Мы сбрасываем ваше состояние на %s"
     :descriptions "%s описаний"
     :all-scheduled-description-by-group "Файл со всеми запланированными описаниями по группам:"
-    :yes "да"
-    :no "нет"
+    :yes "yes"
+    :no "no"
```
Comment (ryukzak) [inline src/course_bot/presentation.clj:119]: We don't need to use translation here.

Comment (Ivanio1) [reply src/course_bot/presentation.clj]: I'm trying to fix it this way: 
image

The logic in the general.clj file is implemented in a similar way, but unfortunately this error crashes: 
image

I'm trying for the second day to figure out how to fix it, but I didn't come to anything. Do you know how to fix it?

**Comment by Ivanio1** [comment]
@ryukzak Thanks for your advice. Ready to merge.

File: src/course_bot/presentation.clj [hunk 2]
```diff
@@ -219,13 +219,13 @@
       :approve
       (fn [tx {{id :id} :from text :text} {desc :desc}]
         (cond
-          (= text (tr :pres/yes)) (do (talk/send-text token id (tr :pres/teacher-will-check))
```
Comment (ryukzak) [inline src/course_bot/presentation.clj:222]: :yes and :no keys (in translation) not used in the code. So they should be removed.

**Review by ryukzak — CHANGES_REQUESTED** [review]
Also, can you check other code on the same problem.

File: src/course_bot/presentation.clj (hunk 2 above)
Comment (Ivanio1) [reply src/course_bot/presentation.clj:222]: Fixed it

**Comment by Ivanio1** [comment]
This problem is only in presentation.clj. All fixed. @ryukzak

**Review by ryukzak — APPROVED** [review]

**Pull request merged**